projects/mfinance/index.js

```diff
@@ -0,0 +1,52 @@
+const abi = require("./abi.json");
+const { staking } = require("../helper/staking");
+const { pool2s } = require("../helper/pool2");
+const { addFundsInMasterChef } = require("../helper/masterchef");
+
+const masterChef = "0x342A8A451c900158BA4f1367C55955b5Fbcb7CCe";
```

Comment:
might be better to use masterchefExports here, that way the list lpPools2Addresses will remain always up to date

Reply:
yeah, is the rigt option, but I tried to use it and I've got something wrong; then I used addFundsToMasterchef and I added the pools2 manually. I think it didn't allow me because of the poolInfo is different or something like that.